modules/dashboard/modules/widgets/base/platforms/grafana/widget.tf

```diff
@@ -141,7 +151,7 @@ locals {
       y = var.coordinates.y
     }
     title = var.name
-    type  = try(local.type_map[var.view], local.type_map.timeSeries)
+    type  = try(local.type_map[var.type], local.type_map.timeSeries)
```

Comment:
can you check and make sure that changing view=>type is applied for all widgets that use this base module?
also it seems that `var.view` is not being used anymore, so it can be removed